src/main/resources/templates/db_scripts/create_job_number_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS ${myuniversity}_${mymodule}.job_number (
+  last_number bigint NOT NULL DEFAULT 1,
+  type varchar(255) NOT NULL
+);
+INSERT INTO ${myuniversity}_${mymodule}.job_number (last_number, type)
+SELECT 1, 'Logs'
```

Comment:
What do you think about this name: FUND_BULK_UPDATES_LOGS?

Reply:
make sense, using FUND_UDPATE_LOGS to identical with enpoint name